react/README.md

```diff
@@ -456,12 +456,39 @@
           {props.items.map((item, index) => (
             <Item
               key={item.key}
-              onClick={() => doSomethingWith(item.name, index)}
+              onClick={() => doSomethingWith(item.name, index)} 
             />
           ))}
         </ul>
       );
     }
+    // We shouldnot use arrow function onClick as it create instance of method doSomethingWith every time in for loop in your example. //Instead we can function call with ==> and access the attribute or pass on as a props to Item React componenet. Let me know if have any other question
+    function ItemList(props) {
+      return (
+        <ul>
+          {props.items.map((item, index) => (
+            <Item
+              key={item.key}
+              onItemClick={this.doSomethingWith} 
```

Comment:
this is an SFC (stateless functional component) - it has no `this`. What is `this.doSomethingWith` here?